0-bootstrap/README.md

```diff
@@ -87,6 +87,7 @@ Also make sure that you've done the following:
       Cloud organization.
    -  The `roles/billing.admin` role on the billing account.
    -  The `roles/resourcemanager.folderCreator` role.
+   -  The `roles/orgpolicy.policyAdmin` role.
```

Comment:
```suggestion
   -  The `roles/orgpolicy.policyAdmin` role on the Google Cloud organization.
```

I would also put it together with the other role that needs to be at the  Google Cloud organization (the first one).

Reply:
done